statsmodels/sandbox/stats/multicomp.py

```diff
@@ -61,20 +62,36 @@
 
 
 '''
-from statsmodels.compat.python import lzip, lrange
-
 import copy
 import math
+from collections import defaultdict
 
 import numpy as np
+import networkx as nx
```

Comment:
networkx is not a dependency of statsmodels.

Reply:
Good catch.  I forgot to add that/bring this up.

Reply:
number 4 is fine, just import in the function
and make unit test conditional on availability of nx. 
We do the same for matplotlib, where we have mostly only smoke test for plot functions and plot methods.
cvxopt is another optional dependency that has imports and unit tests conditional on availability